app/views/groups/_form.html.erb

```diff
@@ -2,6 +2,10 @@
   <%= f.input :title %>
   <%= f.input :description %>
   <%= f.input :summary, as: :trix_editor %>
+
```

Comment:
Maybe use the route to load the project in the controller when instantiating a new group rather than a conditional in the view, i.e. `/projects/<project-slug>/groups/new` (nested routes may be worth a look for this).